setup.py

```diff
@@ -27,7 +27,7 @@ def process_ignore_tags(buffer):
             x for x in buffer.split("\n") if "<!-- ignore_ppi -->" not in x
         )
     try:
-        fpath = os.path.join(os.path.dirname(__file__), "README.md")
+        fpath = os.path.join(os.path.dirname(__file__), "README")
```

Comment:
Personally I'd prefer to keep `README.md` here and just remove `README` from the repository. Looking at 4f71027fcdcb1e4ca8d7e04987588d2f7e615f4b it seems it was added in 2016 and back then `setuptools` didn't add `README.md` to sdists automatically. These days `setuptools` can handle `README.md` just fine if `README` isn't present. Removing `README` can in theory break something (even though in practice it shouldn't) so this patch is somewhat more backward-compatible.

Reply:
I went ahead and removed README.